packages/agoric-cli/test/agops-vaults-smoketest.sh

```diff
@@ -31,9 +31,22 @@ set -x
 
 # open a vault
 OFFER=$(mktemp -t agops.XXX)
-bin/agops vaults open >|"$OFFER"
+bin/agops vaults open --wantMinted 5.00 --giveCollateral 9.0 >|"$OFFER"
 jq ".body | fromjson" <"$OFFER"
 agoric wallet send --keyring-backend="test" --from "$WALLET" --offer "$OFFER"
 
 # list my vaults
 bin/agops vaults list --keyring-backend="test" --from "$WALLET"
+
+# adjust
+OFFER=$(mktemp -t agops.XXX)
+bin/agops vaults adjust --giveCollateral 1.0 --from "$WALLET" --vaultId vault2 >|"$OFFER"
```

Comment:
To get this to work I had to change `vault2` to `vault1` and add `--keyring-backend="test"`
```
bin/agops vaults adjust --keyring-backend="test" --giveCollateral 1.0 --from "$WALLET" --vaultId vault1 >|"$OFFER"
```